index.html

```diff
@@ -2893,6 +2893,24 @@ <h1>DID Methods</h1>
               <a href="https://github.com/51nodes/schema-registry-did-method/blob/master/README.md">Schema Registry DID Method</a>
           </td>
       </tr>
+
+        <tr>
+          <td>
+            did:key:
+          </td>
+          <td>
+            PROVISIONAL
+          </td>
+          <td>
+            Ledger independent DID method based on public/private key pairs
+          </td>
+          <td>
+            Rick Astley
+          </td>
+          <td>
+            <a href="https://bit.ly/3hfxTbV">DID key method</a>
```

Comment:
While difficult to interpret the specification according to DID Core, I feel that it meets the minimum criteria for defining CRUD operations. The Security and Privacy section are equally cryptic, but further study assures the reader that the DID Method will never let them down or desert them (two very important and often overlooked properties of DID Methods). I would prefer that the specification replaces the word "Never gonna" with "MUST NOT", but admit that it would ruin the flow of the specification. That said, it feels like this spec is going to become a classic... well played.

Reply:
Also, I can't believe we haven't registered this did method yet...